web/styles/settings.css

```diff
@@ -853,11 +853,16 @@ input[type="checkbox"] {
         &:disabled {
             cursor: not-allowed;
             background-color: hsl(0deg 0% 93%);
-            opacity: 0.7;
         }
     }
 }
 
+#org-notifications {
+    .dropdown-widget-button {
+        color: hsl(0deg 0% 33%);
```

Comment:
Should this also have CSS when the button is disabled like we have above?

Reply:
Also, the "Default code block lanugage" uses the dropdown-widget, so might be good to update that as well.